omnigibson/object_states/open.py

```diff
@@ -76,37 +77,33 @@ def _get_relevant_joints(obj):
             - list of int: Joint directions for each corresponding relevant joint
     """
     global m
-    if not hasattr(obj, "metadata"):
-        return None, None, None
 
-    both_sides = obj.metadata[m.BOTH_SIDES_METADATA_FIELD] if m.BOTH_SIDES_METADATA_FIELD in obj.metadata else False
+    default_both_sides = False
```

Comment:
nit -- shouldn't we put these lines within the if clause in L85, since the default values aren't used elsewhere?

Reply:
it's used in two different places actually.